src/test/java/shop/VirtualItemTest.java

```diff
@@ -0,0 +1,35 @@
+package shop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class VirtualItemTest {
+
+    VirtualItem testVirtualItem;
+    private static final double SIZE = 999;
+
+    @BeforeEach
+    void setUp() {
+        testVirtualItem = new VirtualItem();
+    }
+
+    @Test
+    @Tag("Virtual_item_set_and_get_size")
+    @Tag ("Smoke")
+    @DisplayName("Set size and get size")
+    void setSizeOnDisk() {
+        double sizeBefore = testVirtualItem.getSizeOnDisk();
+        testVirtualItem.setSizeOnDisk(SIZE);
+        double sizeAfter = testVirtualItem.getSizeOnDisk();
+
+        assertAll("Weigth",
```

Comment:
VirtualItem class is designed as simple model class, which doesn't have any business logic. It doesn't make sense to develop tests for getters/setters. The only method that can be covered with test - toString() method (because class overrides it).
This task has 2 correct resolutions:
- don't write tests at all for this class
- write test for toString method

Reply:
I decided to write a test for toString method. 
commit 365eca85948d73df370cedced112b7f33aaeb410